intranet/apps/files/views.py

```diff
@@ -328,6 +328,109 @@ def can_access_path(fsdir):
 
 
 @login_required
+def files_delete(request, fstype=None):
+    filepath = request.GET.get("dir", None)
```

Comment:
This needs to use POST, not GET. GET should never be used for destructive operations.